### PR TITLE
Update and rename ThirdPartyNotices to ThirdPartyNotices.md

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

**Comment by mairaw** [comment]
Thanks @Youssef1313. I'm marking with the blocked label for now while we wait for the inquiry we sent to our legal team about this and changes like https://github.com/dotnet/docs/pull/15352.

**Comment by Thraka** [comment]
@mairaw Can this move forward?

File: ThirdPartyNotices.md [hunk 1]
```diff
@@ -12,4 +13,4 @@ Microsoft's general trademark guidelines can be found at http://go.microsoft.com
 Privacy information can be found at https://privacy.microsoft.com/en-us/
```
Comment (mairaw) [inline ThirdPartyNotices.md]:
```suggestion
Privacy information can be found at https://privacy.microsoft.com/
```

**Comment by mairaw** [comment]
Yes @Thraka. Just added a suggestion for the change that was made in the legal repo too.

**Pull request merged**